README.md

```diff
@@ -180,22 +180,30 @@ The example performs the following workflow locally:
 
 In order to execute the sample study simply execute from the root directory of the repository:
 
-    $ maestro ./samples/lulesh/lulesh_sample1.yaml
+    $ maestro run ./samples/lulesh/lulesh_sample1.yaml
```

Comment:
You might want to note the macosx version here just in case. `sed` behaves differently between Unix and Mac systems, so they had to be separate specs.

Reply:
Oh, I think is misunderstood. I think you are asking for the macosx spec to also be documented here. The sample spec names have changed. I'll update that.